### DISCUSSION TITLE
ISO 15897 format of country specific locales

### DISCUSSION THREAD

**Discussion by ideepblue** [wintercms/winter#345] [Ideas]
On https://laravel.com/docs/6.x/localization, it mentions:

> For languages that differ by territory, you should name the language directories according to the ISO 15897. For example, "en_GB" should be used for British English rather than "en-gb".

Also, https://github.com/Laravel-Lang/lang/issues/1269 has followed this rule.

Will WinterCMS adopts this rule?

**Comment by bennothommo** [comment]
It's something we can consider, but I'm not sure of the ramifications of switching it to the ISO 15897 format since we've traditionally not closely followed that format.

**Comment by ideepblue** [comment]
That's all right. I also found different software using a different format. It's hard to say which format should follow.
